core/src/test/java/org/elasticsearch/search/sort/FieldSortIT.java

```diff
@@ -857,7 +860,9 @@ public void testSortMissingStrings() throws IOException {
     }
 
     public void testIgnoreUnmapped() throws Exception {
-        createIndex("test");
+        Version version = VersionUtils.randomVersionBetween(random(), Version.V_2_0_0, Version.CURRENT);
+        prepareCreate("test")
+            .setSettings(Settings.builder().put(indexSettings()).put("index.version.created", version).build()).get();
```

Comment:
Can you duplicate this test into one that tests 2.x indices and the other one that tests 5.x indices so that we have 100% coverage with every test run?